tests/test_logging.py

```diff
@@ -1,17 +0,0 @@
-import unittest
-from unittest import mock
-
-from frontstage.logger_config import logger_initial_config
-
-
-class LoggingTestCase(unittest.TestCase):
-    """Test case for logging"""
-
-    def test_basic_logger_config(self):
-        """Test logger configuration"""
-        with mock.patch('logging.basicConfig') as loggingConfig:
-            logger_initial_config(service_name='ras-secure-message', log_level='INFO', logger_format="message", logger_date_format='2017-06-13')
-            loggingConfig.assert_called_with(level='INFO', format="message", datefmt='2017-06-13')
-
-if __name__ == '__main__':
-    unittest.main()
```

Comment:
Probably worth leaving it there and having a look at some refactoring, I'm not sure what it's doing though.